composer.json

```diff
@@ -22,6 +22,7 @@
         "illuminate/http": "^8.0",
         "illuminate/support": "^8.0",
         "illuminate/validation": "^8.0",
+        "laravel/fortify": "^1.7",
```

Comment:
This will pull in the Fortify for all users, which is not want we want. Let's remove this. It isn't needed to check on the `LoginResponse` class.